cmd/ingress-operator/render.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
+)
+
+func render(dir string, prefix string) error {
+	files := []string{
+		manifests.CustomResourceDefinitionManifest,
+		manifests.NamespaceManifest,
+	}
+
+	if dir == "-" {
+		for _, file := range files {
+			fmt.Printf("# file: %s\n", filepath.Base(file))
+			fmt.Println(manifests.MustAssetString(file))
+			fmt.Println("---")
```

Comment:
Write `---` before, and (optionally) `...` after each file.

Reply:
Fixed